TrafficSimulator/Assets/AutoDrive/AutoDrive.cs

```diff
@@ -49,8 +49,19 @@ private enum Status
             Repositioning
         }
 
+        // Used for road registration
+        public delegate void RoadChangedDelegate(Road newRoad);
+        public RoadChangedDelegate OnRoadChanged;
+        
         [Header("Connections")]
-        public Road Road;
+        [SerializeField]
```

Comment:
Should be on the same row as the variable

Reply:
Fixed